ocs_ci/ocs/resources/storage_cluster.py

```diff
@@ -498,3 +499,118 @@ def get_all_storageclass():
         )
     ]
     return storageclass
+
+
+def check_rebalance_occur_after_expand(old_osds):
```

Comment:
not sure if related, but please check if ocs_ci.ocs.cluster.CephCluster.get_rebalance_status can fit here somehow

Reply:
I'm not sure that the function is really checking the balance. It is checking that all pg are active and clean which I would suggest changing it to pg per osd and there it would be closer to rebalance. 
Anyway as long as pg are not active and clean ceph health will be in warn state which occur in ceph_health_check.
Beside that I'm not waiting for final balance to occur.
I will try to write a test with a function that really check the balance status (~same pg number per osd and same ratio of data per osd).